swan-lake/integration-tutorials/service-orchestration.md

```diff
@@ -200,57 +199,73 @@ Follow the instructions given in this section to develop the service.
         decimal discounted;
         string paymentID;
         string status;
-    |};
+    };
     ```
 
-    - The `ReservationRequest` record uses [record type inclusion](https://ballerina.io/learn/by-example/type-inclusion-for-records/) in the type of the `patient` field to include all the fields from the `Patient` record along with the `cardNo` field.
+    > **Note:**
+    > While it is possible to work with the JSON payload directly, using record types offers several advantages including enhanced type safety, data validation, and better tooling experience (e.g., completion).
 
-    - The initial record definitions can be generated using the "Paste JSON as record" VS Code command with the relevant JSON payloads and the records can then be modified as necessary.
+    > **Note:** 
+    > When the fields of the JSON objects are expected to be exactly those specified in the sample payload, the generated records can be updated to be [closed records](https://ballerina.io/learn/by-example/controlling-openness/), which would indicate that no other fields are allowed or expected.
 
-6. Define the [HTTP service (REST API)](https://ballerina.io/learn/by-example/#rest-service) that has the resource that accepts user requests, makes calls to the backend services to retrieve relevant details, and responds to the client.
+4. Define the [HTTP service (REST API)](https://ballerina.io/learn/by-example/#rest-service) that has the resource that accepts user requests, makes calls to the backend services to retrieve relevant details, and responds to the client.
```

Comment:
Let's update "makes calls to the backend services to retrieve relevant details" here also.